config/webpack.prod.config.js

```diff
@@ -45,12 +65,12 @@ if (process.env.ENABLE_NEW_RELIC !== 'false') {
     agentID: process.env.NEW_RELIC_AGENT_ID || 'undefined_agent_id',
     trustKey: process.env.NEW_RELIC_TRUST_KEY || 'undefined_trust_key',
     licenseKey: process.env.NEW_RELIC_LICENSE_KEY || 'undefined_license_key',
-    applicationID: process.env.NEW_RELIC_APP_ID || 'undefined_application_id',
+    applicationID: envConfig.NEW_RELIC_APP_ID || process.env.NEW_RELIC_APP_ID || 'undefined_application_id',
```

Comment:
[curious] Would we expect the other `process.env.NEW_RELIC_*` env vars might be usable with `envConfig` as well (e.g., `agentID`, `trustKey`, `licenseKey`, etc.)?

Reply:
I actually went ahead and removed these changes as this would've meant using a `require` statement in this config to fetch the environment variables, and that got in the way of using `import` in the `env.config.js` file. 

This will mean that there's no clear solution yet for if we ever want to replace the `process.env.ENV_VAR`s in this file or `webpack.dev.config.js`.